docs: Custom App development by non-admins (#3145)
commercetools/merchant-center-application-kit: website/src/content/getting-started.mdx

By default the starter template is configured to use the [HTTP APIs in the GCP-EU Region](https://docs.commercetools.com/api/general-concepts#regions). If you plan to run the Custom Application in another environment, change the `cloudIdentifier` accordingly (see [list of cloud identifiers](/concepts/merchant-center-api#cloud-identifiers)).

> ## Custom Application local development

Reviewer: Wondering if something like this would be more clear?

```suggestion
## User access for local development
```

To start developing your Custom Application locally, the quickest way is by being part of the `Administrators` Team of your Organization. In that case, choose a Project that is part of an Organization where you are an administrator of.

<Info>

> As of version `22.5.0`, an alternative option is available for local development and login. This option is specifically tailored for users belonging to teams with assigned granular permissions. Detailed information about this feature is outlined in the [help needed](/help-needed#granular-permissions-for-custom-application-local-development) section.

Reviewer 2: ```suggestion
As of version `22.5.0`, you have the option of assigning granular permissions to teams to enable local development and login. For more information, see [Granular permissions for local development](/help-needed#granular-permissions-local-development).
```
